### DISCUSSION TITLE
Custom Colours Not Switching Using Theme Changer On Website

### DISCUSSION THREAD

**Discussion by scarcekoi** [tailwindlabs/tailwindcss#15335] [Help]
I'm using the Catppuccin colours but they are not switching. I'm pretty sure this is something with the tailwind.css.
Here is the repository: https://github.com/scarcekoi/website

**Comment by wongjn** [comment]
Could you provide some concrete examples of where it is not working please?

**Reply by scarcekoi** [reply]
I did this: https://github.com/scarcekoi/website/commit/eca1ed560df0991d9d658b0ae94df36fc9057478
and when I switched the theme on the website, it just changed the text.

**Reply by scarcekoi** [reply]
![2](https://github.com/user-attachments/assets/7a678172-6971-4829-8eae-8119f9d20da9)
![1](https://github.com/user-attachments/assets/d8a0f5ae-3470-430a-9e8d-be2b0f5ab96b)

**Reply by wongjn** [reply]
This is how it looks for me locally:
![image](https://github.com/user-attachments/assets/4677ef5f-732b-4b06-8cd3-51a2420902f3)
![image](https://github.com/user-attachments/assets/f0771c5f-d3b0-48f3-992c-2656fd898acc)

**Reply by scarcekoi** [reply]
I reverted the changes so currently it has the original colours and the switching works.
